test.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+import unittest
+import os
+import dotenv
+
+from blagues_api import BlaguesAPI, BlagueType, Blague
+
+dotenv.load_dotenv()
+
+TOKEN = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoiMzI0OTQxNDcyMzY3NjQwNTk2IiwibGltaXQiOjEwMCwia2V5IjoiOHJ2QVdPaEhzdHJlNFRPMFA0MXNZRlMzZTVuajA3QnJOOER2eUwyVFhvczRWUGZ0a0UiLCJjcmVhdGVkX2F0IjoiMjAyMS0wNi0wOVQxNzoyMzozMiswMDowMCIsImlhdCI6MTYyMzI1OTQxMn0.QyqzFGEigCz0yd0K_HJkk8lIABrpI9UDf3UVfSkDIsc" #os.getenv('MY_ENV_VAR')
+
+blagues = BlaguesAPI(TOKEN)
+
+class WrapperTests(unittest.TestCase):
+
+    def test_token(self):
+        """ Check token existance in .env """
+        self.assertIsInstance(TOKEN, str)
+    
+    def test_random_joke(self):
+        """ Check random joke """
+        loop = asyncio.get_event_loop()
+        reponce = loop.run_until_complete(blagues.random())

```

Comment:
Ce n'est pas la peine d'utiliser asyncio, c'est build-in, suffit de mettre async devant la fonction